packages/vite/src/node/__tests__/scan.spec.ts

```diff
@@ -90,7 +90,8 @@ describe('optimizer-scan:script-test', () => {
     const shouldFailArray = [
       `testMultiline("import", {
         body: "ok" });`,
-      `import type, {foo} from 'vue'`
+      `import type, {foo} from 'vue'`,
```

Comment:
This is valid typescript syntax and should not fail, the test is incorrectly expecting it to fail.